wkcuber/api/View.py

```diff
@@ -47,7 +47,15 @@ def write(self, data, relative_offset=(0, 0, 0)):
             sum(x) for x in zip(self.global_offset, relative_offset)
         )
 
-        absolute_offset, data = self._handle_compressed_write(absolute_offset, data)
+        if self._is_compressed():
+            if allow_compressed_write:
+                absolute_offset, data = self._handle_compressed_write(
+                    absolute_offset, data
+                )
+            else:
+                raise ValueError(
+                    "Cannot write unaligned compressed data. Align the data first or use the parameter 'allow_compressed_write', which might impact the performance."
+                )
```

Comment:
Here we don't know yet if the data is unaligned, right? I think we should do nothing in this case